src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -36,10 +37,11 @@
   <DebuggeeBuildProcess>sdk.prebuilt</DebuggeeBuildProcess>
   <DebuggeeBuildRoot>$(RootBinDir)</DebuggeeBuildRoot>
 
-  <!-- Use the global.json SDK to build and the test SDK/runtime to run -->
-  <CliPath>$(RepoRootDir)/.dotnet/dotnet</CliPath>
+ <!-- Use the global.json SDK to build and the test SDK/runtime to run -->
```

Comment:
The comment is incorrect now.  We are using the .dotnet-test SDK to build the tests now.